test/index.ts

```diff
@@ -137,13 +148,14 @@ defineOverridableClient('SubscriberClient');
 
 describe('PubSub', () => {
   // tslint:disable-next-line variable-name
-  let PubSub;
+  let PubSub: typeof pubsubTypes.PubSub;
   const PROJECT_ID = 'test-project';
-  let pubsub;
+  // tslint:disable-next-line no-any
+  let pubsub: pubsubTypes.PubSub;
   const OPTIONS = {
     projectId: PROJECT_ID,
     promise: {},
-  };
+  } as pubsubTypes.ClientConfig;
```

Comment:
nitpick/question: why use `as` instead of

```js
const OPTIONS: pubsubTypes.ClientConfig = {
  // ...
};
```

Reply:
in that way giving error for promise member "Type '{}' is missing the following properties from type 'PromiseConstructor'"